bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-SymbolicName: org.jspecify.jspecify
+Bundle-Name: jspecify
```

Comment:
How about to make it more human readable by `JSpecify annotations`  
Also pom.xml 0.3.0 states: `<name>JSpecify annotations</name>`

Reply:
Thanks for the review. Added your suggestion @bjmi